apps/docs/src/.vitepress/components/DesignVariable.vue

```diff
@@ -97,10 +102,17 @@ const emit = defineEmits<{
     &--no-value {
       padding: var(--onyx-spacing-3xs) var(--onyx-spacing-2xs);
     }
+    &--no-padding {
+      padding: 0;
+    }
   }
 
   &__value {
+    color: var(--onyx-color-base-neutral-500);
     font-weight: 600;
+    font-size: 0.8125rem;
+    border-top: 1px solid var(--onyx-color-base-neutral-300);
```

Comment:
```suggestion
    border-top: var(--onyx-1px-in-rem) solid var(--onyx-color-base-neutral-300);
```

We want to always use rem